src/main/resources/assets/dragonsurvival/lang/en_us.json

```diff
@@ -1339,21 +1339,21 @@
   "ds.advancements.transplant_chorus_fruit.title": "Favorite Treat Everywhere",
   "ds.advancements.transplant_chorus_fruit.description": "Transplant a tasty purple flower back home on the Overworld.",
   "ds.advancements.meat_eater.title": "Meat Eater",
-  "ds.advancements.meat_eater.description": "Eat all unique food that is marked as forest dragon food from Dragon Survival. JEI is your friend!",
+  "ds.advancements.meat_eater.description": "Eat all unique forest dragon food from Dragon Survival.",
   "ds.advancements.master_all_forest_passives.title": "Photosynthesis",
   "ds.advancements.master_all_forest_passives.description": "Achieve total mastery of your forest dragon passive skills.",
   "ds.advancements.be_sea_dragon.title": "Heterochromia Iridum",
   "ds.advancements.be_sea_dragon.description": "Become a sea dragon by any means necessary.",
   "ds.advancements.be_old_sea_dragon.title": "True Ancient Guardian",
   "ds.advancements.be_old_sea_dragon.description": "Gain level 60 growth as a sea dragon.",
   "ds.advancements.loot_shipwreck.title": "Ocean Reclaimer",
-  "ds.advancements.loot_shipwreck.description": "Claim salvage from a 3 ship chest that was lost at sea.",
+  "ds.advancements.loot_shipwreck.description": "Claim salvage from all the different chests in a shipwreck.",
   "ds.advancements.rain_dancing.title": "Dancing in the Rain",
-  "ds.advancements.rain_dancing.description": "Feel hydration during rain and thunderstorm.",
+  "ds.advancements.rain_dancing.description": "Get hydrated from the rain!",
```

Comment:
It is very important to specify that it must be rain AND thunderstorm Because you need both.

Reply:
Oh, I thought you just had to be in any rain at all, is this how the advancement actually works?

Reply:
Yes, I specifically put “and” in there.

Reply:
The requirements for this advancement are to be exposed to _either_ rain or thunderstorm conditions.  It it not necessary to do both.

Reply:
For advancements, it's a list of lists, with each list requiring you to meet at least one condition within that sub-list.
`  "requirements": [
    [
      "raining",
      "thundering"
    ],
    [
      "is_sea_dragon"
    ]
  ],`
So, "is_sea_dragon" is always required to be met, while either of "raining" or "thundering" will work.

Reply:
Okay, so there was some random bug and I couldn't get that advancement with just rain. Now I tested again and only rain works.